apps/src/templates/MarketingSkinnyBanner.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import DCDO from '@cdo/apps/dcdo';
+
+// Not using inline styles to keep regular and responsive styles together.
+//   Copied styles from 'shared/css/csa_skinny_banner.scss' that's used
+//   in the Pegasus version of this banner:
+//  'pegasus/sites.v3/code.org/views/csa_skinny_banner.haml'.
+import './marketingSkinnyBanner.scss';
+
+const IMAGE_BASE_URL = '/blockly/media/';
+
+// Skinny banner component
+// Current: CSA Launch May 2022
```

Comment:
Nit: no need to comment that this is the skinny banner component, that is clear from the context. I'm curious why you included the second comment? An alternative would be to name the component `CsaLaunchSkinnyBanner` to more clearly describe the purpose

Reply:
@maureensturgeon I wasn't sure if we'd want to reuse this component with another feature/product launch in the future; now that we can do this Mktg might want to use it more. Do you think it's better to do one-off banners instead?

Reply:
I think if you pass in the info that's specific to the CSA launch through props, it's ok to give the banner a generic name, but since the content is hard-coded it makes more sense to name the banner specifically what it's used for. And then if you change the hard-coded content in the future, also change the name of the banner

Reply:
This is a bit of a nit, it's ok to leave the name as-is, if that makes the most sense to you

Reply:
I'll do this one-off banner for now and can refactor to a generic version in the future if needed. Thanks for all the great feedback! 🙏 🎉